app/views/anthologies/show.html.erb

```diff
@@ -17,6 +17,7 @@
       <% end %>
       <div class="media-left">
         <% unless @anthology.banner.blank? %>
+
           <%= link_to image_tag(@anthology.banner.url(:medium), class: 'media-object'), @anthology.banner.url(:medium), target: '_blank' %>
```

Comment:
What happens if the uploaded image is smaller than the dimensions of the `:medium` version? Is a medium version created?